apisix/plugins/loggly.lua

```diff
@@ -85,6 +86,17 @@ local schema = {
             type = "boolean",
             default = true
         },
+        severity_map = {
+            type = "object",
+            description = "upstream response code vs syslog severity mapping",
+            patternProperties = {
+                [".*"] = {
```

Comment:
We can use regex to verify the range of status code?

Reply:
Yea, I had tested. It doesn't work that way. 
say for eg,
`["^[1-5][0-9]{2}"] = {...}` the check schema match the enums for only the set of keys where the key is a status code (regex). For other scenarios (key = "ABC", "500a") there is no matching regex, so it simply doesn't match and validate with the pattern and let it pass as valid inputs.

Reply:
Does `["^[1-5][0-9]{2}$"] = {...}` work for it?

Reply:
@spacewander It works as well as doesn't work.
Example 1: Want :heavy_check_mark:  Got: :heavy_check_mark: 
```json
...
           "severity_map": {
                "200": "INFO"
            }
...
```

Example 2: Want :x:  Got: :x: 
```json
...
           "severity_map": {
                "200": "abc"
            }
...
```

Example 3: Want :x:  Got: :heavy_check_mark: : 
```json
...
           "severity_map": {
                "abc": "def"
            }
...
```
The problem is, if the key fails to match regex, the pattern properties don't get applied. Also, the program doesn't throw an error for the key mismatch.

Reply:
Got it. What about adding `"additionalProperties": false`? So the mismatch key will cause an error.

Reply:
It worked! Awesome : )
```
// inp "severity_map": {
//               "abc": "def"
//            }
{
    "error_msg": "failed to check the configuration of plugin loggly err: property \"severity_map\" validation failed: additional properties forbidden, found abc"
}
```